web/src/api/ws.jsx

```diff
@@ -80,6 +80,12 @@ export function useWs(watchTopic, publishTopic) {
 
   const send = useCallback(
     (payload, retain = false) => {
+      // if ws is closed but user has interacted with page
+      // refresh to ensure state is up to date
+      if (ws.readyState == WebSocket.CLOSED) {
+        location.reload();
```

Comment:
Is there not a way to try reconnecting the websocket without a refresh?

Reply:
I couldn't figure out how to do that because ws object does not have that function directly and the send function doesn't have access to the fields necessary to rebuild the ws object. 

I believe refreshing may provide some utility in the case that frigate has stopped but the user tries to interact with the webUI. 

That being said, I believe @hawkeye217 may have a better way to fix this

Reply:
I've used [`react-use-websocket`](https://github.com/robtaussig/react-use-websocket) before, it's pretty simple and robust. But I'm not sure it has preact compatibility... so unless we want to go down the react-compat route, my fix may not be as straightforward as I would have hoped.